Do not use github.sha in pull_request and workflow_run (#5)
kolonialno/upload-velo-artifacts: README.md

  A GitHub Action to upload Velo deployment artifacts
</p>

> Upload-velo-artifacts is part of the Velo deploy system at Oda. It first checkouts to `sha` and then uploads `deploy_folder`.

Reviewer: ```suggestion
`upload-velo-artifacts` is part of the Velo deploy system at Oda. It first performs a checkout of `sha` and then uploads `deploy_folder`.
```
nitpick: language/formatting
